### DISCUSSION TITLE
CSV file on Python

### DISCUSSION THREAD

**Discussion by Charlineprg** [community/community#21498] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,</p>
<p>I have to write a program which processes a csv file with all the names given in France since the 19th century. I want to separate the list in two. In the first, I want to put the boy’s name and in the second, I want to put the girl’s name. I tried many things but they have not worked. Could someone help me to find a solution please ?</p>
<p>Thanks in advance</p>
<pre><code>import csv

# création de listes vides pour les données
liste = []
masculin = []
feminin = []

# Ouverture du fichier
with open("nat2018-groupe-NSI.csv", 'r', encoding ='UTF-8', newline = '') as f:
    liste_dicos = csv.reader(f)
    for prenom in liste_dicos :
        if ['sexe'] == '1' :
            masculin.append(prenom)
        else :
            feminin.append(prenom)
</code></pre>

**Comment by fire-eggs** [comment]
<p>No one can help you if you don’t provide a portion of the CSV file so we can tell what you are trying to read. Also, you don’t tell us how your program “fails”.</p>

**Comment by Charlineprg** [comment]
<pre><code>sexe;preusuel;annee;nombre
1;ALEXANDRE;1900;977
1;ALEXANDRE;1901;1086
1;ALEXANDRE;1902;1071
1;ALEXANDRE;1903;1063
1;ALEXANDRE;1904;1076
1;ALEXANDRE;1905;1165
1;ALEXANDRE;1906;1053
1;ALEXANDRE;1907;1044
1;ALEXANDRE;1908;1054
1;ALEXANDRE;1909;1076
1;ALEXANDRE;1910;1129
1;ALEXANDRE;1911;1064
1;ALEXANDRE;1912;1081
1;ALEXANDRE;1913;973
1;ALEXANDRE;1914;965
1;ALEXANDRE;1915;668
1;ALEXANDRE;1916;528
1;ALEXANDRE;1917;481
1;ALEXANDRE;1918;538
1;ALEXANDRE;1919;633
1;ALEXANDRE;1920;1001
1;ALEXANDRE;1921;882
1;ALEXANDRE;1922;852
1;ALEXANDRE;1923;839
1;ALEXANDRE;1924;734
1;ALEXANDRE;1925;795
1;ALEXANDRE;1926;757
1;ALEXANDRE;1927;683
1;ALEXANDRE;1928;649
1;ALEXANDRE;1929;641
1;ALEXANDRE;1930;664
1;ALEXANDRE;1931;626
1;ALEXANDRE;1932;583
1;ALEXANDRE;1933;514
1;ALEXANDRE;1934;406
1;ALEXANDRE;1935;509
1;ALEXANDRE;1936;430
1;ALEXANDRE;1937;384
1;ALEXANDRE;1938;336
1;ALEXANDRE;1939;335
1;ALEXANDRE;1940;297
1;ALEXANDRE;1941;226
1;ALEXANDRE;1942;241
1;ALEXANDRE;1943;245
1;ALEXANDRE;1944;232
1;ALEXANDRE;1945;209
1;ALEXANDRE;1946;299
1;ALEXANDRE;1947;303
1;ALEXANDRE;1948;283
1;ALEXANDRE;1949;254
1;ALEXANDRE;1950;265
1;ALEXANDRE;1951;243
1;ALEXANDRE;1952;232
1;ALEXANDRE;1953;227
1;ALEXANDRE;1954;261
1;ALEXANDRE;1955;229
1;ALEXANDRE;1956;223
1;ALEXANDRE;1957;254
1;ALEXANDRE;1958;270
1;ALEXANDRE;1959;321
1;ALEXANDRE;1960;299
1;ALEXANDRE;1961;322
1;ALEXANDRE;1962;342
1;ALEXANDRE;1963;437
1;ALEXANDRE;1964;490
1;ALEXANDRE;1965;628
1;ALEXANDRE;1966;737
1;ALEXANDRE;1967;837
1;ALEXANDRE;1968;1046
1;ALEXANDRE;1969;1291
1;ALEXANDRE;1970;1826
1;ALEXANDRE;1971;2139
1;ALEXANDRE;1972;3084
1;ALEXANDRE;1973;3202
1;ALEXANDRE;1974;3407
1;ALEXANDRE;1975;4069
1;ALEXANDRE;1976;4486
1;ALEXANDRE;1977;4937
1;ALEXANDRE;1978;5213
1;ALEXANDRE;1979;6158
1;ALEXANDRE;1980;6992
1;ALEXANDRE;1981;7027
1;ALEXANDRE;1982;7195
1;ALEXANDRE;1983;6321
1;ALEXANDRE;1984;6074
1;ALEXANDRE;1985;6931
1;ALEXANDRE;1986;8742
1;ALEXANDRE;1987;8218
1;ALEXANDRE;1988;8253
1;ALEXANDRE;1989;8326
1;ALEXANDRE;1990;8373
1;ALEXANDRE;1991;8883
1;ALEXANDRE;1992;8775
1;ALEXANDRE;1993;8956
1;ALEXANDRE;1994;8830
1;ALEXANDRE;1995;9015
1;ALEXANDRE;1996;8867
1;ALEXANDRE;1997;8193
1;ALEXANDRE;1998;7515
1;ALEXANDRE;1999;6959
1;ALEXANDRE;2000;6380
1;ALEXANDRE;2001;5544
1;ALEXANDRE;2002;5035
1;ALEXANDRE;2003;5102
1;ALEXANDRE;2004;4157
1;ALEXANDRE;2005;4082
1;ALEXANDRE;2006;3845
1;ALEXANDRE;2007;3347
1;ALEXANDRE;2008;2916
1;ALEXANDRE;2009;2772
1;ALEXANDRE;2010;2609
1;ALEXANDRE;2011;2325
1;ALEXANDRE;2012;2103
1;ALEXANDRE;2013;1970
1;ALEXANDRE;2014;1814
1;ALEXANDRE;2015;1732
1;ALEXANDRE;2016;1571
1;ALEXANDRE;2017;1411
1;ALEXANDRE;2018;1378
1;BASTIEN;1900;3
1;BASTIEN;1901;7
1;BASTIEN;1902;3
1;BASTIEN;1903;4
1;BASTIEN;1904;7
1;BASTIEN;1905;6
1;BASTIEN;1907;3
1;BASTIEN;1908;3
1;BASTIEN;1909;10
1;BASTIEN;1910;5
1;BASTIEN;1911;5
1;BASTIEN;1912;6
1;BASTIEN;1914;6
1;BASTIEN;1915;4
1;BASTIEN;1916;3
1;BASTIEN;1917;5
1;BASTIEN;1918;6
1;BASTIEN;1919;4
1;BASTIEN;1920;4
1;BASTIEN;1921;5
1;BASTIEN;1922;6
1;BASTIEN;1923;3
1;BASTIEN;1924;6
1;BASTIEN;1925;4
1;BASTIEN;1926;4
1;BASTIEN;1927;8
1;BASTIEN;1928;3
1;BASTIEN;1929;4
1;BASTIEN;1930;3
1;BASTIEN;1931;5
1;BASTIEN;1935;4
1;BASTIEN;1939;5
1;BASTIEN;1945;4
1;BASTIEN;1947;4
1;BASTIEN;1948;3
1;BASTIEN;1949;3
1;BASTIEN;1950;3
1;BASTIEN;1951;4
1;BASTIEN;1953;3
1;BASTIEN;1954;3
1;BASTIEN;1955;4
1;BASTIEN;1957;4
1;BASTIEN;1958;4
1;BASTIEN;1963;4
1;BASTIEN;1964;6
1;BASTIEN;1965;3
1;BASTIEN;1966;8
1;BASTIEN;1967;16
1;BASTIEN;1968;20
1;BASTIEN;1969;14
1;BASTIEN;1970;22
1;BASTIEN;1971;49
1;BASTIEN;1972;82
1;BASTIEN;1973;86
1;BASTIEN;1974;145
1;BASTIEN;1975;167
1;BASTIEN;1976;153
1;BASTIEN;1977;226
1;BASTIEN;1978;333
1;BASTIEN;1979;370
1;BASTIEN;1980;495
1;BASTIEN;1981;449
1;BASTIEN;1982;678
1;BASTIEN;1983;675
1;BASTIEN;1984;675
1;BASTIEN;1985;898
1;BASTIEN;1986;1039
1;BASTIEN;1987;1191
1;BASTIEN;1988;1384
1;BASTIEN;1989;1385
1;BASTIEN;1990;1919
1;BASTIEN;1991;1827
1;BASTIEN;1992;1791
1;BASTIEN;1993;1703
1;BASTIEN;1994;2048
1;BASTIEN;1995;1979
1;BASTIEN;1996;1892
1;BASTIEN;1997;1976
1;BASTIEN;1998;2380
1;BASTIEN;1999;2388
1;BASTIEN;2000;2193
1;BASTIEN;2001;2005
1;BASTIEN;2002;1788
1;BASTIEN;2003;1815
1;BASTIEN;2004;1558
1;BASTIEN;2005;1547
1;BASTIEN;2006;1369
1;BASTIEN;2007;1201
1;BASTIEN;2008;1177
1;BASTIEN;2009;1026
1;BASTIEN;2010;1064
1;BASTIEN;2011;1204
1;BASTIEN;2012;967
1;BASTIEN;2013;1000
1;BASTIEN;2014;852
1;BASTIEN;2015;802
1;BASTIEN;2016;644
1;BASTIEN;2017;750
1;BASTIEN;2018;572
1;BASTIEN;XXXX;27
2;CHARLINE;1900;8
2;CHARLINE;1901;31
2;CHARLINE;1902;18
2;CHARLINE;1903;13
2;CHARLINE;1904;13
2;CHARLINE;1905;16
2;CHARLINE;1906;23
2;CHARLINE;1907;25
2;CHARLINE;1908;13
2;CHARLINE;1909;17
2;CHARLINE;1910;21
2;CHARLINE;1911;18
2;CHARLINE;1912;38
2;CHARLINE;1913;18
2;CHARLINE;1914;46
2;CHARLINE;1915;41
2;CHARLINE;1916;17
2;CHARLINE;1917;32
2;CHARLINE;1918;23
2;CHARLINE;1919;44
2;CHARLINE;1920;76
2;CHARLINE;1921;86
2;CHARLINE;1922;80
2;CHARLINE;1923;81
2;CHARLINE;1924;94
2;CHARLINE;1925;75
2;CHARLINE;1926;80
2;CHARLINE;1927;105
2;CHARLINE;1928;105
2;CHARLINE;1929;121
2;CHARLINE;1930;112
2;CHARLINE;1931;143
2;CHARLINE;1932;120
2;CHARLINE;1933;129
2;CHARLINE;1934;126
2;CHARLINE;1935;122
2;CHARLINE;1936;130
2;CHARLINE;1937;127
2;CHARLINE;1938;114
2;CHARLINE;1939;138
2;CHARLINE;1940;127
2;CHARLINE;1941;92
2;CHARLINE;1942;109
2;CHARLINE;1943;124
2;CHARLINE;1944;109
2;CHARLINE;1945;131
2;CHARLINE;1946;203
2;CHARLINE;1947;186
2;CHARLINE;1948;167
2;CHARLINE;1949;179
2;CHARLINE;1950;184
2;CHARLINE;1951;195
2;CHARLINE;1952;161
2;CHARLINE;1953;156
2;CHARLINE;1954;170
2;CHARLINE;1955;144
2;CHARLINE;1956;132
2;CHARLINE;1957;130
2;CHARLINE;1958;128
2;CHARLINE;1959;111
2;CHARLINE;1960;101
2;CHARLINE;1961;89
2;CHARLINE;1962;68
2;CHARLINE;1963;67
2;CHARLINE;1964;58
2;CHARLINE;1965;51
2;CHARLINE;1966;51
2;CHARLINE;1967;35
2;CHARLINE;1968;41
2;CHARLINE;1969;39
2;CHARLINE;1970;41
2;CHARLINE;1971;50
2;CHARLINE;1972;46
2;CHARLINE;1973;54
2;CHARLINE;1974;55
2;CHARLINE;1975;77
2;CHARLINE;1976;67
2;CHARLINE;1977;83
2;CHARLINE;1978;121
2;CHARLINE;1979;134
2;CHARLINE;1980;171
2;CHARLINE;1981;230
2;CHARLINE;1982;343
2;CHARLINE;1983;362
2;CHARLINE;1984;454
2;CHARLINE;1985;705
2;CHARLINE;1986;759
2;CHARLINE;1987;860
2;CHARLINE;1988;933
2;CHARLINE;1989;1033
2;CHARLINE;1990;1042
2;CHARLINE;1991;1018
2;CHARLINE;1992;916
2;CHARLINE;1993;797
2;CHARLINE;1994;810
2;CHARLINE;1995;808
2;CHARLINE;1996;850
2;CHARLINE;1997;735
2;CHARLINE;1998;728
2;CHARLINE;1999;671
2;CHARLINE;2000;725
2;CHARLINE;2001;695
2;CHARLINE;2002;582
2;CHARLINE;2003;582
2;CHARLINE;2004;556
2;CHARLINE;2005;504
2;CHARLINE;2006;553
2;CHARLINE;2007;537
2;CHARLINE;2008;581
2;CHARLINE;2009;634
2;CHARLINE;2010;634
2;CHARLINE;2011;601
2;CHARLINE;2012;598
2;CHARLINE;2013;582
2;CHARLINE;2014;592
2;CHARLINE;2015;549
2;CHARLINE;2016;530
2;CHARLINE;2017;501
2;CHARLINE;2018;462
1;ÉLOUAN;2010;10
1;ÉLOUAN;2011;30
1;ÉLOUAN;2012;32
1;ÉLOUAN;2013;27
1;ÉLOUAN;2014;20
1;ÉLOUAN;2015;25
1;ÉLOUAN;2016;20
1;ÉLOUAN;2017;21
1;ÉLOUAN;2018;34
1;ÉLOUAN;XXXX;3
1;ELOUAN;1974;4
1;ELOUAN;1975;3
1;ELOUAN;1979;6
1;ELOUAN;1980;6
1;ELOUAN;1981;5
1;ELOUAN;1982;6
1;ELOUAN;1983;7
1;ELOUAN;1984;13
1;ELOUAN;1985;4
1;ELOUAN;1986;6
1;ELOUAN;1987;6
1;ELOUAN;1988;6
1;ELOUAN;1989;16
1;ELOUAN;1990;8
1;ELOUAN;1991;11
1;ELOUAN;1992;10
1;ELOUAN;1993;18
1;ELOUAN;1994;19
1;ELOUAN;1995;27
1;ELOUAN;1996;27
1;ELOUAN;1997;58
1;ELOUAN;1998;75
1;ELOUAN;1999;116
1;ELOUAN;2000;162
1;ELOUAN;2001;242
1;ELOUAN;2002;293
1;ELOUAN;2003;388
1;ELOUAN;2004;453
1;ELOUAN;2005;471
1;ELOUAN;2006;579
1;ELOUAN;2007;563
1;ELOUAN;2008;485
1;ELOUAN;2009;483
1;ELOUAN;2010;447
1;ELOUAN;2011;396
1;ELOUAN;2012;362
1;ELOUAN;2013;301
1;ELOUAN;2014;243
1;ELOUAN;2015;222
1;ELOUAN;2016;172
1;ELOUAN;2017;188
1;ELOUAN;2018;134
2;ENZO;1997;3
2;ENZO;1998;10
2;ENZO;1999;8
2;ENZO;2000;4
2;ENZO;2012;3
2;ENZO;XXXX;30
1;ENZO;1923;6
1;ENZO;1925;12
1;ENZO;1926;4
1;ENZO;1927;5
1;ENZO;1929;5
1;ENZO;1930;4
1;ENZO;1931;3
1;ENZO;1932;3
1;ENZO;1933;3
1;ENZO;1935;3
1;ENZO;1936;4
1;ENZO;1944;4
1;ENZO;1949;9
1;ENZO;1950;5
1;ENZO;1951;10
1;ENZO;1952;6
1;ENZO;1953;13
1;ENZO;1954;8
1;ENZO;1955;6
1;ENZO;1956;6
1;ENZO;1957;13
1;ENZO;1958;17
1;ENZO;1959;22
1;ENZO;1960;30
1;ENZO;1961;26
1;ENZO;1962;19
1;ENZO;1963;18
1;ENZO;1964;26
1;ENZO;1965;24
1;ENZO;1966;16
1;ENZO;1967;23
1;ENZO;1968;20
1;ENZO;1969;12
1;ENZO;1970;15
1;ENZO;1971;12
1;ENZO;1972;6
1;ENZO;1973;13
1;ENZO;1974;13
1;ENZO;1975;7
1;ENZO;1976;6
1;ENZO;1977;3
1;ENZO;1978;4
1;ENZO;1979;5
1;ENZO;1980;16
1;ENZO;1981;7
1;ENZO;1982;14
1;ENZO;1983;5
1;ENZO;1984;6
1;ENZO;1985;11
1;ENZO;1986;19
1;ENZO;1987;21
1;ENZO;1988;56
1;ENZO;1989;106
1;ENZO;1990;146
1;ENZO;1991;253
1;ENZO;1992;376
1;ENZO;1993;361
1;ENZO;1994;565
1;ENZO;1995;700
1;ENZO;1996;953
1;ENZO;1997;1285
1;ENZO;1998;1887
1;ENZO;1999;2790
1;ENZO;2000;3956
1;ENZO;2001;4675
1;ENZO;2002;5962
1;ENZO;2003;6747
1;ENZO;2004;7965
1;ENZO;2005;8852
1;ENZO;2006;8325
1;ENZO;2007;7493
1;ENZO;2008;6980
1;ENZO;2009;6491
1;ENZO;2010;5950
1;ENZO;2011;5410
1;ENZO;2012;4684
1;ENZO;2013;4179
1;ENZO;2014;3738
1;ENZO;2015;3099
1;ENZO;2016;2768
1;ENZO;2017;2589
1;ENZO;2018;2070
1;ENZO;XXXX;20
1;PIERRE;1900;7456
1;PIERRE;1901;8449
1;PIERRE;1902;8978
1;PIERRE;1903;9074
1;PIERRE;1904;9215
1;PIERRE;1905;9429
1;PIERRE;1906;9855
1;PIERRE;1907;10074
1;PIERRE;1908;10769
1;PIERRE;1909;10893
1;PIERRE;1910;11156
1;PIERRE;1911;10986
1;PIERRE;1912;11910
1;PIERRE;1913;12208
1;PIERRE;1914;12295
1;PIERRE;1915;7767
1;PIERRE;1916;6060
1;PIERRE;1917;6746
1;PIERRE;1918;7788
1;PIERRE;1919;8655
1;PIERRE;1920;15643
1;PIERRE;1921;15727
1;PIERRE;1922;15085
1;PIERRE;1923;15521
1;PIERRE;1924;15714
1;PIERRE;1925;16102
1;PIERRE;1926;16250
1;PIERRE;1927;16389
1;PIERRE;1928;16302
1;PIERRE;1929;15920
1;PIERRE;1930;16484
1;PIERRE;1931;16082
1;PIERRE;1932;15697
1;PIERRE;1933;14884
1;PIERRE;1934;14673
1;PIERRE;1935;14161
1;PIERRE;1936;13437
1;PIERRE;1937;12865
1;PIERRE;1938;12453
1;PIERRE;1939;11891
1;PIERRE;1940;9986
1;PIERRE;1941;9025
1;PIERRE;1942;9677
1;PIERRE;1943;9863
1;PIERRE;1944;9316
1;PIERRE;1945;8599
1;PIERRE;1946;12394
1;PIERRE;1947;12164
1;PIERRE;1948;11253
1;PIERRE;1949;10800
1;PIERRE;1950;10317
1;PIERRE;1951;9231
1;PIERRE;1952;8750
1;PIERRE;1953;8228
1;PIERRE;1954;7901
1;PIERRE;1955;7655
1;PIERRE;1956;7473
1;PIERRE;1957;7409
1;PIERRE;1958;7305
1;PIERRE;1959;7171
1;PIERRE;1960;6857
1;PIERRE;1961;6510
1;PIERRE;1962;6062
1;PIERRE;1963;6211
1;PIERRE;1964;5756
1;PIERRE;1965;5643
1;PIERRE;1966;5324
1;PIERRE;1967;4896
1;PIERRE;1968;4754
1;PIERRE;1969;4229
1;PIERRE;1970;3753
1;PIERRE;1971;3650
1;PIERRE;1972;3259
1;PIERRE;1973;3030
1;PIERRE;1974;2609
1;PIERRE;1975;2417
1;PIERRE;1976;2447
1;PIERRE;1977;2686
1;PIERRE;1978;3216
1;PIERRE;1979;3613
1;PIERRE;1980;3966
1;PIERRE;1981;3968
1;PIERRE;1982;4294
1;PIERRE;1983;4266
1;PIERRE;1984;4343
1;PIERRE;1985;4745
1;PIERRE;1986;4840
1;PIERRE;1987;5182
1;PIERRE;1988;5568
1;PIERRE;1989;5072
1;PIERRE;1990;5249
1;PIERRE;1991;5458
1;PIERRE;1992;5041
1;PIERRE;1993;4951
1;PIERRE;1994;4822
1;PIERRE;1995;5137
1;PIERRE;1996;4991
1;PIERRE;1997;4665
1;PIERRE;1998;4285
1;PIERRE;1999;4380
1;PIERRE;2000;4160
1;PIERRE;2001;3617
1;PIERRE;2002;2914
1;PIERRE;2003;2650
1;PIERRE;2004;2618
1;PIERRE;2005;2281
1;PIERRE;2006;2005
1;PIERRE;2007;1790
1;PIERRE;2008;1645
1;PIERRE;2009;1400
1;PIERRE;2010;1233
1;PIERRE;2011;1137
1;PIERRE;2012;1075
1;PIERRE;2013;996
1;PIERRE;2014;971
1;PIERRE;2015;789
1;PIERRE;2016;745
1;PIERRE;2017;654
1;PIERRE;2018;588
1;RÉMI;1900;92
1;RÉMI;1901;107
1;RÉMI;1902;118
1;RÉMI;1903;122
1;RÉMI;1904;146
1;RÉMI;1905;131
1;RÉMI;1906;167
1;RÉMI;1907;145
1;RÉMI;1908;176
1;RÉMI;1909;167
1;RÉMI;1910;155
1;RÉMI;1911;138
1;RÉMI;1912;176
1;RÉMI;1913;180
1;RÉMI;1914;164
1;RÉMI;1915;97
1;RÉMI;1916;81
1;RÉMI;1917;105
1;RÉMI;1918;117
1;RÉMI;1919;106
1;RÉMI;1920;199
1;RÉMI;1921;210
1;RÉMI;1922;250
1;RÉMI;1923;278
1;RÉMI;1924;252
1;RÉMI;1925;292
1;RÉMI;1926;331
1;RÉMI;1927;321
1;RÉMI;1928;340
1;RÉMI;1929;337
1;RÉMI;1930;394
1;RÉMI;1931;325
1;RÉMI;1932;378
1;RÉMI;1933;369
1;RÉMI;1934;380
1;RÉMI;1935;385
1;RÉMI;1936;346
1;RÉMI;1937;393
1;RÉMI;1938;350
1;RÉMI;1939;306
1;RÉMI;1940;285
1;RÉMI;1941;231
1;RÉMI;1942;264
1;RÉMI;1943;266
1;RÉMI;1944;286
1;RÉMI;1945;346
1;RÉMI;1946;471
1;RÉMI;1947;445
1;RÉMI;1948;496
1;RÉMI;1949;521
1;RÉMI;1950;529
1;RÉMI;1951;544
1;RÉMI;1952;577
1;RÉMI;1953;574
1;RÉMI;1954;625
1;RÉMI;1955;627
1;RÉMI;1956;653
1;RÉMI;1957;723
1;RÉMI;1958;710
1;RÉMI;1959;841
1;RÉMI;1960;766
1;RÉMI;1961;762
1;RÉMI;1962;768
1;RÉMI;1963;768
1;RÉMI;1964;717
1;RÉMI;1965;703
1;RÉMI;1966;805
1;RÉMI;1967;621
1;RÉMI;1968;774
1;RÉMI;1969;703
1;RÉMI;1970;572
1;RÉMI;1971;605
1;RÉMI;1972;605
1;RÉMI;1973;498
1;RÉMI;1974;483
1;RÉMI;1975;560
1;RÉMI;1976;540
1;RÉMI;1977;701
1;RÉMI;1978;717
1;RÉMI;1979;799
1;RÉMI;1980;974
1;RÉMI;1981;1082
1;RÉMI;1982;1876
1;RÉMI;1983;2240
1;RÉMI;1984;2352
1;RÉMI;1985;2232
1;RÉMI;1986;2536
1;RÉMI;1987;2616
1;RÉMI;1988;2446
1;RÉMI;1989;2476
1;RÉMI;1990;2481
1;RÉMI;1991;2560
1;RÉMI;1992;2413
1;RÉMI;1993;2117
1;RÉMI;1994;2099
1;RÉMI;1995;2006
1;RÉMI;1996;2173
1;RÉMI;1997;2133
1;RÉMI;1998;2078
1;RÉMI;1999;1980
1;RÉMI;2000;1952
1;RÉMI;2001;1806
1;RÉMI;2002;1602
1;RÉMI;2003;1431
1;RÉMI;2004;1250
1;RÉMI;2005;1058
1;RÉMI;2006;936
1;RÉMI;2007;813
1;RÉMI;2008;759
1;RÉMI;2009;660
1;RÉMI;2010;335
1;RÉMI;2011;274
1;RÉMI;2012;345
1;RÉMI;2013;274
1;RÉMI;2014;223
1;RÉMI;2015;195
1;RÉMI;2016;186
1;RÉMI;2017;146
1;RÉMI;2018;152
2;ZOÉ;1900;131
2;ZOÉ;1901;146
2;ZOÉ;1902;138
2;ZOÉ;1903;158
2;ZOÉ;1904;124
2;ZOÉ;1905;130
2;ZOÉ;1906;113
2;ZOÉ;1907;105
2;ZOÉ;1908;109
2;ZOÉ;1909;123
2;ZOÉ;1910;104
2;ZOÉ;1911;95
2;ZOÉ;1912;81
2;ZOÉ;1913;95
2;ZOÉ;1914;97
2;ZOÉ;1915;62
2;ZOÉ;1916;50
2;ZOÉ;1917;29
2;ZOÉ;1918;38
2;ZOÉ;1919;33
2;ZOÉ;1920;64
2;ZOÉ;1921;58
2;ZOÉ;1922;42
2;ZOÉ;1923;59
2;ZOÉ;1924;55
2;ZOÉ;1925;43
2;ZOÉ;1926;49
2;ZOÉ;1927;41
2;ZOÉ;1928;37
2;ZOÉ;1929;36
2;ZOÉ;1930;34
2;ZOÉ;1931;27
2;ZOÉ;1932;34
2;ZOÉ;1933;24
2;ZOÉ;1934;36
2;ZOÉ;1935;30
2;ZOÉ;1936;20
2;ZOÉ;1937;16
2;ZOÉ;1938;9
2;ZOÉ;1939;18
2;ZOÉ;1940;16
2;ZOÉ;1941;12
2;ZOÉ;1942;14
2;ZOÉ;1943;14
2;ZOÉ;1944;11
2;ZOÉ;1945;3
2;ZOÉ;1946;6
2;ZOÉ;1947;7
2;ZOÉ;1948;10
2;ZOÉ;1949;5
2;ZOÉ;1950;8
2;ZOÉ;1951;11
2;ZOÉ;1952;8
2;ZOÉ;1953;3
2;ZOÉ;1954;9
2;ZOÉ;1955;9
2;ZOÉ;1956;5
2;ZOÉ;1958;7
2;ZOÉ;1959;6
2;ZOÉ;1960;8
2;ZOÉ;1961;12
2;ZOÉ;1962;9
2;ZOÉ;1963;5
2;ZOÉ;1964;10
2;ZOÉ;1965;12
2;ZOÉ;1966;6
2;ZOÉ;1967;9
2;ZOÉ;1968;12
2;ZOÉ;1969;6
2;ZOÉ;1970;11
2;ZOÉ;1971;13
2;ZOÉ;1972;18
2;ZOÉ;1973;12
2;ZOÉ;1974;19
2;ZOÉ;1975;9
2;ZOÉ;1976;14
2;ZOÉ;1977;17
2;ZOÉ;1978;23
2;ZOÉ;1979;15
2;ZOÉ;1980;30
2;ZOÉ;1981;35
2;ZOÉ;1982;39
2;ZOÉ;1983;50
2;ZOÉ;1984;61
2;ZOÉ;1985;88
2;ZOÉ;1986;112
2;ZOÉ;1987;166
2;ZOÉ;1988;191
2;ZOÉ;1989;238
2;ZOÉ;1990;290
2;ZOÉ;1991;338
2;ZOÉ;1992;346
2;ZOÉ;1993;341
2;ZOÉ;1994;368
2;ZOÉ;1995;428
2;ZOÉ;1996;582
2;ZOÉ;1997;758
2;ZOÉ;1998;1025
2;ZOÉ;1999;1242
2;ZOÉ;2000;1420
2;ZOÉ;2001;1861
</code></pre>
<p>The program is executed but it puts all the names in the male and female list as if there were no conditions. And so I end up with 2 lists containing all the names. Maybe I can convert the first and last character to an integer but I don’t know how to take it.</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>You want to read about using csv.DictReader instead of csv.reader.</p>
<p>With DictReader you will get a dictionary which is indexed based on the column names in the first line of the file. Then you will be able to do things like:</p>
<pre><code>if prenom['sexe'] == '1':
</code></pre>
<p>You can’t do that with the regular csv.reader.</p>
<p>Your “csv” file is actually using a semi-colon as the delimiter. It may be necessary to let the csv package know this, by including </p>
<pre><code>delimiter=';'
</code></pre>
<p>as a parameter to the csv.DictReader call.</p>

**Comment by Charlineprg** [comment]
<p>That work. Thank you </p>
